README.md

```diff
@@ -49,9 +51,9 @@ By default, the config file will be loaded from ~/.oraclebmc/config. Alternate l
 
 ## Setting the Compartment
 
-Each BMCS command requires a compartment ID, which will default to the root compartment. If you do not have the correct permissions and you do not specify a different compartment, then you will receive an authorization error.
+Each BMCS command requires a compartment ID, which will default to the root compartment. If you do not have the correct permissions then you will receive an authorization error.
```

Comment:
I'd be in favor of keeping the origin text for the "setting the compartment" section, with the one change to say "Most BMCS commands require" instead of "Each BMCS command requires".